data/input_files/tests/thermal_conduction/.static_amgx_solve.lua

```diff
@@ -1,3 +1,9 @@
+-- meemee Failing: error message:
+--[[
+ERROR (/usr/WS2/meemee/serac/repo/src/serac/numerics/equation_solver.cpp:147)]
+AMGX was not enabled when MFEM was built
```

Comment:
Please undo the renaming and remove this comment.  You should log an issue that these need to be added as conditional integration tests instead (which also needs to be implemented).